src/DetailsView/Styles/detailsview.scss

```diff
@@ -395,10 +395,14 @@ div.insights-file-issue-details-dialog-container {
             .dark-gray {
                 color: $neutral-60;
             }
+            .is-selected .button-flex-container {
```

Comment:
1. Yes, these changes hurt me personally to make 😢 
2. I did verify/validate that the non-reflow UI components did not have modified behavior as a result.